frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx

```diff
@@ -181,14 +181,14 @@ const BackingStorePage: React.FC<BackingStorePageProps> = React.memo(
         <Form className="nb-bc-step-page-form">
           <Title headingLevel="h3" size="xl" className="nb-bc-step-page-form__title">
             <Flex breakpointMods={[{ modifier: 'justify-content-space-between' }] as any}>
-              <FlexItem>Tier 1 - backingStore ({state.tier1Policy})</FlexItem>
+              <FlexItem>Tier 1 - backing Store ({state.tier1Policy})</FlexItem>
```

Comment:
```suggestion
              <FlexItem>Tier 1 - Backing Store ({state.tier1Policy})</FlexItem>
```

Reply:
Yes, the labels should start with a capital letter, the same as other entities in the console (such as Storage Class).